workers/scanner_test.go

```diff
@@ -1,14 +1,183 @@
 package workers
 
 import (
+	"context"
 	"testing"
 	"time"
 
+	"github.com/skynetlabs/pinner/conf"
 	"github.com/skynetlabs/pinner/skyd"
 	"github.com/skynetlabs/pinner/test"
+	"gitlab.com/NebulousLabs/errors"
+	"gitlab.com/SkynetLabs/skyd/build"
 	"gitlab.com/SkynetLabs/skyd/skymodules"
 )
 
+const (
+	// cyclesToWait establishes a common number of sleepBetweenScans cycles we
+	// should wait until we consider that a file has been or hasn't been picked
+	// by the scanner.
+	cyclesToWait = 5
+)
+
+var (
+	// maxSleepBetweenScans is the maximum time we might sleep between scans.
+	maxSleepBetweenScans = time.Duration(float64(sleepBetweenScans) * (1 + sleepVariationFactor))
```

Comment:
The tests in this file were moved here because we no longer distinguish between integration and other long tests and it is better to have the tests closer and access unexported variables than exporting those variables for the sole reason that the tests need them.